packages/priority-queue/src/main/Item.ts

```diff
@@ -15,14 +15,19 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
-
 import Priority from './Priority';
 
 export default class Item {
+  /** original business logic */
   fn: Function = () => {};
+  label: string | undefined;
   priority: number = Priority.MEDIUM;
+  /** wrapped `reject` of `fn` */
   reject: Function = () => {};
+  /** wrapped `resolve` of `fn` */
   resolve: Function = () => {};
-  retry: number | undefined = 0;
+  /** number of retries for rejecting Promises */
+  retry: number = Infinity;
+  /** time when the item has been added to the queue */
```

Comment:
Is that some sort of special comment? Can we just use `//`?

Reply:
It's JSDoc, hover over the variable in VSCode to see the description.